src/sagemaker/jumpstart/factory/model.py

```diff
@@ -565,6 +569,14 @@ def _add_config_name_to_kwargs(kwargs: JumpStartModelInitKwargs) -> JumpStartMod
             kwargs.config_name or specs.inference_configs.get_top_config_from_ranking().config_name
         )
 
+        if kwargs.config_name:
+            resolved_config = specs.inference_configs.configs[kwargs.config_name].resolved_config
```

Comment:
is it ever possible to have key error here?

Reply:
I can add a more indicative error message when the config name is invalid (not in the keys)

Reply:
I'm ok with this now since it's private beta, but let's make sure these are accounted for in GA

Reply:
This is already included in the `types.py`, added a test case though